docker/docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
       - GP2GP_SSL_TRUST_STORE_URL
       - GP2GP_SSL_TRUST_STORE_PASSWORD
       - GP2GP_GPC_OVERRIDE_NHS_NUMBER
+      - GP2GP_GPC_OVERRIDE_FROM_ASID
+      - GP2GP_GPC_OVERRIDE_TO_ASID
```

Comment:
Need to update README to include these

Reply:
Also consider: we do actually need both of these? We only use `GP2GP_GPC_OVERRIDE_TO_ASID` right?

Reply:
It seems that SSP in Int is stricter than it used to be. Good to know, we may actually need to use these in production.